medical_compliance/medical_compliance/api/store_utils.py

```diff
@@ -115,7 +111,8 @@ def insert_measurement(store_endpoint_host_uri, user_id, device_id,
 
         "user": user_uri,
         "device": device_uri,
-        "value_info": value_info
+        "value_info": value_info,
+        "ok": True
```

Comment:
Great! Please remember to file a follow-up ticket on this so we don't loose sight of it, thank you!